export-exchanges.js

```diff
@@ -36,9 +36,16 @@ function replaceInFile (filename, regex, replacement) {
 
 // ---------------------------------------------------------------------------
 
+const approvedIds = fs.readFileSync ('.ccxt.approve.cfg')
```

Comment:
I hope you don't mind if I edit this a little, also, I'd switch the config to the json format, ok?